### PR TITLE
Adding many MDO hunting queries to Defender XDR solution

### PR DESCRIPTION
**Pull request by damozes1**
Change(s):
   - Added 86+ MDO related hunting queries to the Defender XDR solutions Hunting section. https://github.com/Azure/Azure-Sentinel/tree/master/Hunting%20Queries/Microsoft%20365%20Defender/Email%20Queries. 
   - Added 2 new MDO related queries as well under the folder Hunting cover new use cases. 

Reason for Change(s):
   - Very few MDO-email security focused queries were present in the solution. We have many more under community queries now so closing the gap.

 Checked that the validations are passing and have addressed any issues that are present:
These AH queries already used as Community queries and were validated they working queries based on real world use cases from many sources

### CODE REVIEW

File: Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/Remediation/Email remediation action list.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+id: 99713387-9d61-49eb-8edc-f51153d8bb01
+name: Listing Email Remediation Actions via Explorer
+description: |
+  Listing Email Remediation Actions perfomed via Explorer in Defender for Offie 365
+description-detailed: |
+  Listing Email Remediation Actions perfomed via Explorer in Defender for Offie 365
```
Comment (rahul0216) [inline Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/Remediation/Email remediation action list.yaml:6]: Spelling mistakes 'perfomed' and 'Offie'

File: Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/URL click on ZAP Email.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+id: efe27064-6d35-4720-b7f5-e0326695613d
+name: URL click on ZAP email
+description: |
+  In this query, we looking for Url clicks on emails wich get actioned by Zerohour auto purge
+description-detailed: |
+  In this query, we looking for Url clicks on emails wich get actioned by Zerohour auto purge (ZAP) in Defender for Office 365.
```
Comment (rahul0216) [inline Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/URL click on ZAP Email.yaml:6]: Grammer and spelling mistakes 'we looking' and 'wich'

File: Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/URL clicks actions by URL.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+id: bc46e331-3cb0-483d-9c90-989d2a59457f
+name: URL clicks actions by URL
+description: |
+  In this query, we looking URL click actions by URL in th elast 7 days
+description-detailed: |
+  In this query, we looking URL click actions by URL in th elast 7 days using Defender for Office 365 data.
```
Comment (rahul0216) [inline Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/URL clicks actions by URL.yaml:6]: Spelling mistake 'th elast'

File: Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/URLClick details based on malicious URL click alert.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+id: 03e61096-20d0-46eb-b8e0-a507dd00a19f
+name: URLClick details based on malicious URL click alert
+description: |
+  In this query, we looking for Url clicks on emails wich are generated the alert-A potentially malicious URL click was detected
+description-detailed: |
+  In this query, we looking for Url clicks on emails wich are generated the alert-A potentially malicious URL click was detected in Defender for Offiec 365.
```
Comment (rahul0216) [inline Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/URLClick details based on malicious URL click alert.yaml:6]: Grammer and spelling mistakes 'we looking' and 'wich'

File: Hunting Queries/Microsoft 365 Defender/Email Queries/URL Click/URL clicks actions by URL.yaml [hunk 5]
```diff
@@ -2,16 +2,21 @@ id: bc46e331-3cb0-483d-9c90-989d2a59457f
 name: URL clicks actions by URL
 description: |
   In this query, we looking URL click actions by URL in th elast 7 days
+description-detailed: |
+  In this query, we looking URL click actions by URL in th elast 7 days using Defender for Office 365 data.
```
Comment (rahul0216) [inline Hunting Queries/Microsoft 365 Defender/Email Queries/URL Click/URL clicks actions by URL.yaml:6]: Spelling mistake 'th elast'

File: Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/User clicks on phishing URLs in emails.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+id: d823da0e-1334-4a66-8ff4-2c2c40d26295
+name: User clicks on phishing URLs in emails
+description: |
+  This query helps in determining clickthroughs when email delivered because of detetcion overrides.
```
Comment (rahul0216) [inline Solutions/Microsoft Defender XDR/Hunting Queries/Email Queries/URL Click/User clicks on phishing URLs in emails.yaml:4]: Spelling error 'detetcion'

**Review by rahul0216 — COMMENTED** [review]
Mostly looks good. Please address the inline comments.

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**